dashboard/src/features/transactions/components/New/New.jsx

```diff
@@ -153,7 +159,7 @@ class Form extends React.Component {
                 <tr>
                   <td><input id='submit_action_submit' type='radio' {...submit_action} value='submit' checked={submit_action.value == 'submit'} /></td>
                   <td>
-                    <label htmlFor='submit_action_submit'>Submit transaction to blockchain</label>
+                    <label htmlFor='submit_action_submit'>Submit transaction to blockchain</label><br></br>
```

Comment:
For tags with no content, let's use `<br />` style tags. Also - can you put this on a new line between the two labels?